stingray/parameterestimation.py

```diff
@@ -0,0 +1,741 @@
+
+__all__ = ["OptimizationResults", "ParameterEstimation", "PSDParEst"]
+
+
+## check whether matplotlib is installed for easy plotting
+try:
+    import matplotlib.pyplot as plt
+    from matplotlib.ticker import MaxNLocator
+    can_plot = True
+except ImportError:
+    can_plot = False
+
+## check whether emcee is installed for sampling
+try:
+    import emcee
+    can_plot = True
+except ImportError:
+    can_sample = False
+
+try:
+    import corner
+    use_corner =True
+except ImportError:
+    use_corner=False
+
+
+#### GENERAL IMPORTS ###
+import numpy as np
+import scipy
+import scipy.optimize
+import scipy.stats
+import scipy.signal
+import copy
+
+
+try:
+    from statsmodels.tools.numdiff import approx_hess
+    comp_hessian = True
+except ImportError:
+    comp_hessian = False
+
+
+### own imports
+from . import PSDPosterior
+
+
+class OptimizationResults(object):
+
+
+    def __init__(self, lpost, res, neg=True):
+        """
+        Helper class that will contain the results of the regression.
+        Less fiddly than a dictionary.
+
+        Parameters
+        ----------
+        lpost: instance of Posterior or one of its subclasses
+            The object containing the function that is being optimized
+            in the regression
+
+
+        res: instance of scipy's OptimizeResult class
+            The object containing the results from a optimization run
+
+        """
```

Comment:
Hi ! I believe this docstring belongs to the class as whole and should be put out of the `__init__` method.